components/bl-signature-pad-component/src/buttons/save-signature-button.js

```diff
@@ -0,0 +1,40 @@
+import { dataURLToBlob } from '../utils/file';
+
+export function SaveSignatureButton({ signaturePad, component, eventHandlers }) {
+  const { saveSignatureButtonVisibility, saveSignatureButtonLabel } = component;
+  const { onSaveSignatureClick } = eventHandlers;
+
+  const saveSignature = () => {
+    if (signaturePad.isEmpty()) {
+      alert('Please provide a signature first.');
+
+      return;
```

Comment:
Why here we have a return, but not in another save actions?

Reply:
No need to save empty signature to Blob, so we have a return.
In other save actions we have save logic in the "else" block.